sqlmesh/dbt/basemodel.py

```diff
@@ -170,17 +170,29 @@ def config_name(self) -> str:
         return f"{self.package_name}.{self.name}"
 
     @property
-    def sql_name(self) -> str:
+    def full_sql_name(self) -> str:
         """
-        Get the sqlmesh model name
+        Get the fully qualified model name
 
         Returns:
-            The sqlmesh model name
+            The fully qualified model name
         """
         return ".".join(
             part for part in (self.database, self.table_schema, self.table_name) if part
         )
 
+    def sql_name(self, context: DbtContext) -> str:
```

Comment:
Should we call this `canonical_name` for clarity?

Reply:
I have no opinion. I'm good with whatever. It used to be called `sqlmesh_name` I believe.

Reply:
IMHO `sql_name` outlived its relevance

Reply:
Up and onwards!